dev-support/git-jira-release-audit/README.md

```diff
@@ -0,0 +1,186 @@
+<!---
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Git / JIRA Release Audit
+
+This is an application for performing an audit between the histories on our git
+branches and the `fixVersion` field set on issues in JIRA. It does this by
+building a Sqlite database from the commits found on each git branch,
+identifying Jira IDs and release tags, and then requesting information about
+those issues from Jira. Once both sources have been collected, queries can be
+performed against the database to look for discrepancies between the sources of
+truth (and, possibly, bugs in this script).
+
+## Setup
+
+The system prerequisites are Python3 with VirtualEnv available and Sqlite. Also,
+you'll need the content of this directory and a local checkout of git repository.
+
+Build a VirtualEnv with the script's dependencies with
+
+```shell script
+$ python3 --version
+Python 3.7.6
+$ python3 -m venv ./venv
```

Comment:
Is this the "recommended" way of doing it? I've always done: `virtualenv venv && source venv/bin/activate` in the past.

Reply:
Dunno what you mean by "recommended". This is python -- there are multiple ways to solve a problem. In this case, I wanted to be sure I was getting virtualenv from a python3 install, hence the `-mvenv` invocation. I also have no need to "drop into" the environment, so `venv/bin/activate` doesn't buy me anything.

Reply:
Oh, virtualenv used to be a completely separate thing and it's been rolled into python 3.3 under the `venv` module (https://virtualenv.pypa.io/en/latest/). I guess this approach eliminates getting some "wrong" virtualenv executable. Just curious, not an issue :)